pkgs/top-level/python-aliases.nix

```diff
@@ -50,6 +50,7 @@ mapAliases ({
   bt_proximity = bt-proximity; # added 2021-07-02
   BTrees = btrees; # added 2023-02-19
   carrot = throw "carrot has been removed, as its development was discontinued in 2012"; # added 2022-01-18
+  cchardet = faust-cchardet; # added 2023-02-14
```

Comment:
On second thought, won't that bite us with regards to packages relying on aliases?

```
❯ rg cchardet -l
pkgs/development/tools/poetry2nix/poetry2nix/overrides/build-systems.json
pkgs/development/python-modules/cchardet/default.nix
pkgs/development/python-modules/aiohttp/default.nix
pkgs/development/python-modules/surepy/default.nix
pkgs/development/python-modules/nextcord/default.nix
pkgs/development/python-modules/vulcan-api/default.nix
pkgs/development/python-modules/clevercsv/default.nix
pkgs/applications/misc/calibre/default.nix
pkgs/servers/home-assistant/appdaemon.nix
pkgs/tools/backup/grab-site/default.nix
pkgs/tools/security/xcat/default.nix
pkgs/tools/security/cve-bin-tool/default.nix
```

Reply:
Yes, I forgot to change the references.